### DISCUSSION TITLE
icons in the editor tab

### DISCUSSION THREAD

**Discussion by af4jm** [gitkraken/vscode-gitlens#1433] [Q&A]
Where is the setting to tell GitLens to never put any icons in the space at the end of the row of tabs?

**Comment by shaman-apprentice** [comment]
Hey, I was wondering this myself just now. Found the solution [here](https://github.com/eamodio/vscode-gitlens/issues/669#issuecomment-467531508). I just created #1445  if we could maybe add this to the documentation.

**Reply by Juarrow** [reply]
For anyone else coming across this:

to your settings.json (or workspace settings file) add
````
"gitlens.menus": {
    editorGroup: false
}
````

And it's gone
